### PR TITLE
Remove Chat v0

### PR DESCRIPTION
**Pull request by philipperolet**
Migration (dropping the 3 chat tables) to be run after merge

### CODE REVIEW

File: front/next.config.js [hunk 1]
```diff
@@ -23,6 +23,11 @@ module.exports = removeImports({
           "https://dust-tt.notion.site/Terms-of-Use-ff8665f52c454e0daf02195ec0d6bafb",
         permanent: true,
       },
+      {
+        source: "/w/:wId/u/chat/:cId",
+        destination: "/w/:wId/assistant/:cId",
+        permanent: true,
```
Comment (spolu) [inline front/next.config.js:29]: let's do permanent false in case we need to revert?

File: front/migrations/20230925_drop_v0_tables.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+DROP TABLE chat_retrieved_documents;
```
Comment (spolu) [inline front/migrations/20230925_drop_v0_tables.sql:1]: Let's not run that just yet in case we need to revert (at least 1 week I think)

**Review by spolu — CHANGES_REQUESTED** [review]
LGTM \o/

Left a minor change.

Can you check that there was no /v1/ API call yesterday? :pray:

**Comment by philipperolet** [comment]
> LGTM \o/
> 
> Left a minor change.
> 
> Can you check that there was no /v1/ API call yesterday? 🙏

:+1: I assumed you meant to v1/w/[wid]/chats (only v1 route suppressed) One failed request (no api key) in the last 2 days so good

**Comment by spolu** [comment]
Awesome

**Review by spolu — DISMISSED** [review]

**Pull request merged**